samples/charts/data-chart/scatter-bubble-chart-fill-scale/index.html

```diff
@@ -17,6 +17,31 @@
     <div id="root">
 
 <div class="container sample">
+    <div class="options vertical">
```

Comment:
I am not sure what's the goal with the sliders here, but they don't seem to do have any effect on the chart